test/org/sagebionetworks/bridge/play/controllers/UserDataDownloadControllerTest.java

```diff
@@ -54,7 +51,7 @@ public void test() throws Exception {
 
         // validate args sent to mock service
         ArgumentCaptor<DateRange> dateRangeCaptor = ArgumentCaptor.forClass(DateRange.class);
-        verify(mockService).requestUserData(eq(studyIdentifier), same(user), dateRangeCaptor.capture());
+        verify(mockService).requestUserData(eq(studyIdentifier), any(String.class), dateRangeCaptor.capture());
```

Comment:
Added participant with email to mockSession and verify that's what is used in call.